Cargo.toml

```diff
@@ -27,14 +27,16 @@ paste = { version = "1.0.3", optional = true }
 pyo3-macros = { path = "pyo3-macros", version = "=0.13.0", optional = true }
 unindent = { version = "0.1.4", optional = true }
 hashbrown = { version = "0.9", optional = true }
+serde = {version = "1.0", optional = true}
 
 [dev-dependencies]
 assert_approx_eq = "1.1.0"
 trybuild = "1.0.23"
 rustversion = "1.0"
 proptest = { version = "0.10.1", default-features = false, features = ["std"] }
 # features needed to run the PyO3 test suite
-pyo3 = { path = ".", default-features = false, features = ["macros", "auto-initialize"] }
+pyo3 = { path = ".", default-features = false, features = ["macros", "auto-initialize", "serde"] }
```

Comment:
Strictly speaking, we don't _need_ this feature to run the tests - they straight up don't work without the other features enabled.

I'd rather just see the `serde` feature added to the list of features tested in `ci.yml`.

Reply:
Thanks for the tip! Added serde to ci.yml